src/test/java/com/aws/greengrass/shadowmanager/ShadowManagerUnitTest.java

```diff
@@ -107,6 +108,7 @@
 class ShadowManagerUnitTest extends GGServiceTestUtil {
```

Comment:
is there a test where both the list and map are set?

Reply:
Yes. `GIVEN_good_sync_configuration_without_nucleus_thing_config_in_map_and_list_WHEN_initialize_THEN_processes_configuration_correctly`